doc/modules/calibration.rst

```diff
@@ -74,10 +74,14 @@ by showing the number of samples in each predicted probability bin.
 
 .. currentmodule:: sklearn.linear_model
 
-:class:`LogisticRegression` returns well calibrated predictions by default as it has a
+:class:`LogisticRegression` is more likely to return well calibrated predictions by itself as it has a
```

Comment:
Maybe to address @lorentzenchr's comment:
```suggestion
:class:`LogisticRegression` is likely to return well calibrated predictions by itself as it has a
```

Reply:
I'd rather not: some people are going to argue that it's not that likely in absolute